bash_scripting_101/bash_scripting_101.md

```diff
@@ -92,21 +107,21 @@ Everything that is necessary to run bash on your Mac or Windows machine is proba
 - Search Bash in Cortana to launch WSL for the first time and click “Y” when asked to install Ubuntu for the first time. [Microsoft has their own internal documentation on WSL which features additional visual resources and FAQS](https://docs.microsoft.com/en-us/windows/wsl/).
```

Comment:
The Windows Installation needs to be updated for users who do not have admin access.